docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3"
+services:
+  valheim:
+    image: sethmachineio/valheim-server
+    user: "1000:1000"
```

Comment:
nit, same as prior, what is the issue with having the user be `steam` (from the base image) ?